### PR TITLE
Revert the changes for NNPACK build issue

### PR DESCRIPTION
**Pull request by Liam-Sturge**
This patch undoes the change that was put in place to prevent the build and installation of NNPACK from failing due to a renaming of the default branch to main by the NNPACK external dependency cpuinfo.

See https://github.com/apache/tvm/issues/13871

The issue has been fixed at the source by PR #214 which is now merged in to NNPACK, so the change to `ubuntu_install_nnpack.sh` is no longer required:

https://github.com/Maratyszcza/NNPACK/pull/214

CC @ashutosh-arm @leandron @driazati @areusch

### CODE REVIEW

File: docker/install/ubuntu_install_nnpack.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ git clone https://github.com/Maratyszcza/NNPACK NNPACK
 git clone https://github.com/Maratyszcza/pthreadpool  NNPACK/pthreadpool
 
 # Use specific versioning tag.
-(cd NNPACK && sed -i 's/GIT_TAG master/GIT_TAG main/g' ./cmake/DownloadCpuinfo.cmake && git checkout 1e005b0c2)
+(cd NNPACK && git checkout 1e005b0c2)
```
Comment (ashutosh-arm) [inline docker/install/ubuntu_install_nnpack.sh:29]: If the old SHA is used, the old branch of `CPUInfo` will get picked again. Old problem of `nnpack` build failure in TVM will reappear. Shouldn't we use the latest nnpack SHA i.e., `70a77f485e8b934224f3a79efd8edcd84cd377b8` that reflects the change to pick up thew new branch name of `CPUInfo`?

Comment (Liam-Sturge) [reply docker/install/ubuntu_install_nnpack.sh]: You are right. Thanks @ashutosh-arm , I have fixed the patch with the latest commit.

**Comment by Liam-Sturge** [comment]
@tvm-bot rerun

**Comment by Liam-Sturge** [comment]
@tvm-bot rerun

**Review by driazati — APPROVED** [review]

**Pull request merged**